.setup/install_submitty/get_globals.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+set -ve
+
+# Get arguments
+for cli_arg in "$@"
+do
+    if [[ $cli_arg =~ ^config=.* ]]; then
+        SUBMITTY_CONFIG_DIR="$(readlink -f "$(echo "$cli_arg" | cut -f2 -d=)")"
+    fi
+done
+
```

Comment:
same as above...
(this script should crash with an appropriate error message if SUBMITTY_CONFIG_DIR is not successfully set)